embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/arch/destination/TelemetryDestinationImpl.kt

```diff
@@ -129,6 +129,7 @@ internal class TelemetryDestinationImpl(
             type = type,
             attributes = attributes,
         )
+        processStateService.sessionUpdated()
```

Comment:
Ideally every function in `TelemetryDestination` should call this. Perhaps that's a good candidate to break off into a separate PR?